LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIMission_LWCustomMission.uc

```diff
@@ -318,6 +328,28 @@ simulated function AddIgnoreButton()
 	}
 }
 
+simulated function OnButtonSizeRealized()
```

Comment:
In my version of this fix I added a comment that explained why this was empty. I can add it in later if you'd like. I like to explain unusual code so that others are aware of gotchas and don't try to prune stuff like this.